dev-vcs/fuel/fuel-1.0.1.recipe

```diff
@@ -0,0 +1,89 @@
+SUMMARY="A QT Fossil GUI"
+DESCRIPTION=" Fuel is cross-platform GUI front-end for the excellent Fossil SCM tool \
+written in Qt. Fuel runs on Windows, OSX as well as various flavors of Unix including \
+Linux and FreeBSD"
```

Comment:
```diff
-DESCRIPTION=" Fuel is cross-platform GUI front-end for the excellent Fossil SCM tool \
-written in Qt. Fuel runs on Windows, OSX as well as various flavors of Unix including \
-Linux and FreeBSD"
+DESCRIPTION="Fuel is cross-platform GUI front-end for the excellent Fossil SCM \
+tool written in Qt. Fuel runs on Windows, OSX as well as various flavors of \
+Unix including Linux and FreeBSD."
```
Changes:
* No space before `Fuel`
* line splits to have lines <= 80 chars
* full-stop after `FreeBSD`

Reply:
omg.. i forget again.. ok